infra/dockerfile.base

```diff
@@ -11,3 +11,4 @@ RUN apt-get update && \
 
 COPY . /tmp/bugbug
 RUN pip install --disable-pip-version-check --no-cache-dir /tmp/bugbug
+RUN rm -rf /tmp/bugbug
```

Comment:
If you rm the directory after another Docker layer, the size of the image won't be affected, so this is not useful.

Reply:
If I use `RUN pip install --disable-pip-version-check --no-cache-dir /tmp/bugbug && rm -rf /tmp/bugbug`

This will not store it. Am I right?

Reply:
No, it will still be stored, as you are storing it in the previous layer.